packages/slice-machine/pages/index.tsx

```diff
@@ -152,7 +93,7 @@ const CustomTypes: React.FunctionComponent = () => {
         }
         MainBreadcrumb={
           <Fragment>
-            <FiLayout /> <Text ml={2}>Custom Types</Text>
+            <MdSpaceDashboard /> <Text ml={2}>Custom Types</Text>
```

Comment:
thsi seems to be duplicated with CTBuilder / layout /...

Reply:
Nop it's not exactly the same, in the CTBuilder, there is the same starting name but ont the CTBuilder / layout /... there is the name of the custom types after it. 

I think it's little bit overkill to create a specific component for 2 lines of code. I mean if we had to create a component every time we used 2 components together we will have many components and that will lead on extra overhead going trought all the component tree. What do you want me to do ?